### PR TITLE
Description of how to apply animated parameters

### LINKED ISSUE #216: Define Bezier curve concretely for the animated parameters

### PR DESCRIPTION
**Pull request by felicialim**
Fixes #216

***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/pull/222.html" title="Last updated on Dec 18, 2022, 12:30 AM UTC (ddbd55d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/222/77f7ada...ddbd55d.html" title="Last updated on Dec 18, 2022, 12:30 AM UTC (ddbd55d)">Diff</a>

### CODE REVIEW

**Comment by sunghee-hwang** [comment]
In STEP mode, does "start_point_value" value mean constant value to be applied to all samples of the segment?

**Comment by felicialim** [comment]
> In STEP mode, does "start_point_value" value mean constant value to be applied to all samples of the segment?

Yes, I've updated to clarify this.

File: index.bs [hunk 1]
```diff
@@ -2091,6 +2094,40 @@ After relevant processing, multiple audio elements are mixed into one channel au
...
+0 <= a <= 1.
+```
+
+If [=animation_type=] is equal to BEZIER, the parameter value is interpolated following a quadratic Bezier curve between [=start_point_value=] and [=end_point_value=] as follows:
+
+```
+B_quad(a) = (1 - a)^2 * P0 + 2 * (1 - a) * a * P1 + a^2 * P2,
+0 <= a <= 1.
+```
+
```
Comment (sunghee-hwang) [inline index.bs:2130]: B_quad(a) is not a value but a coordinate.

Let's B_quad(`a`) = (`x`, `y`), 
1. What is a representative of elements of the segment? `x` . In other words, would `y` be the parameter value of `x`? 
2.  If so, this equation is to let decoders know the parameter value for a given `a` not for a given `x`. Devices need to know the parameter value for a given `x` not for a given `a`.

**Comment by tdaede** [comment]
Waiting for clarification from @felicialim .

**Pull request merged**